### ISSUE TITLE
Don't download subtitles of the media language

### ISSUE THREAD

**Issue by TheSpixxyQ** [morpheus65535/bazarr#306]
Hello,
is it possible to not to download subtitles in language of the media language?
Like if I have set default subtitles language Czech, after I add Czech movie to Radarr, it downloads Czech subtitles - which I don't need. I need them only for English movies.
I don't know, if there is any way now or is it the same as [this](https://github.com/morpheus65535/bazarr/issues/209) issue, so I'm rather asking.
Thank you!

**Comment by halali** [comment]
Neviem si uplne predstavit ako by to mohlo fungovat. v radarr api maju audiolanguage iba filmy ktore su vo formate MKV od kvalitneho ripera... tie ostatne su nezname... cize by to fungovalo iba pri niektorych filmoch a nebolo by to moc spolahlive.

**Comment by morpheus65535** [comment]
We can have a settings that, when enabled, exclude audio language from desired subtitles languages list.

**Comment by TheSpixxyQ** [comment]
Asi to bere z profilu (mám pro EN a CZ zvlášť profily), v Bazarru mám u úplně každého filmu a seriálu napsaný jazyk [screen](https://i.imgur.com/ltttGyQ.jpg)

Yeah, I was thinking about something just like "if wanted subtitle language equals audio language then don't download"

**Comment by halali** [comment]
Hej to bere profile language. Ak stačí to ze sa to bude orientovať podľa profile language tak Také ničo môžem urobiť.

**Comment by TheSpixxyQ** [comment]
Určitě bude, jelikož mám profil nastavený u všeho.
Díky!

**Comment by sharp004** [comment]
Yes that would be great for radarr, but it's seems to be already done for sonarr : in my case, for all my french tv show, I can select "None" for "Subtitles languages", option that I cannot select for Radarr's movies.

But you're right, if it can be automated at the importation of the movie/tv show when the audio language and default subtitle languages are the same, that would be excellent !
